etc/profile-m-z/ssh-agent.profile

```diff
@@ -1,37 +1,62 @@
 # Firejail profile for ssh-agent
+# Description: OpenSSH authentication agent
 # This file is overwritten after every install/update
 quiet
 # Persistent local customizations
 include ssh-agent.local
 # Persistent global definitions
 include globals.local
 
+blacklist ${RUNUSER}/wayland-*
+
 # Allow ssh (blacklisted by disable-common.inc)
 include allow-ssh.inc
 
-blacklist /tmp/.X11-unix
-blacklist ${RUNUSER}/wayland-*
+ignore noexec ${RUNUSER}
 
 include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
 include disable-programs.inc
+include disable-shell.inc
+include disable-X11.inc
+include disable-xdg.inc
 
+include whitelist-common.inc
+include whitelist-run-common.inc
+include whitelist-runuser-common.inc
 include whitelist-usr-share-common.inc
 
+apparmor
 caps.drop all
+machine-id
 netfilter
 no3d
 nodvd
+noinput
+nogroups
 nonewprivs
+noprinters
 noroot
+nosound
 notv
+nou2f
```

Comment:
```suggestion
```

If anything, this would be the quintessential application that uses hardware
keys :p

Reply:
@glitsj16 [on Mar 25](https://github.com/netblue30/firejail/pull/5753#discussion_r1148330269):

> Not according to my observations no. Other parts of OpenSSH need that kind of
> access, like ssh, ssh-add, ssh-copy-id, ssh-key{gen,scan}... The
> authentication agent itself is kept 'dumb' on purpose (which is a good thing
> security-wise). We don't have profiles (yet) for ssh-add and the others
> mentioned.

Sorry, now that you mention it, the hardware keys might only be needed by
ssh-add, not ssh-agent (I'm not sure).

Reply:
I agree. But the way I read its man page (in this case more specifically the `-P` switch) it's actually ssh-add that needs u2f access, the agent only stores whatever is put in. Sadly I don't have access to hardware keys to properly test my assumptions. Does anyone on the collaborator team?

Reply:
Well that's the thing. I'm not sure either. Too many open-ended questions for comfort... Especially the points you refer to below, in combination with @rusty-snake's [hint](https://github.com/netblue30/firejail/issues/5751#issuecomment-1483760338) send me back to the drawing board :)